packages/client/hmi-client/src/components/workflow/tera-workflow.vue

```diff
@@ -301,6 +302,11 @@ const toggleOptionsMenu = (event) => {
 	optionsMenu.value.toggle(event);
 };
 
+const isEditingNote = ref(false);
+const editingNote = (isEditing) => {
+	isEditingNote.value = isEditingNote.value ? isEditing : false;
```

Comment:
What is this test? Why not?
```suggestion
	isEditingNote.value = isEditing ?? false;
```

Reply:
code changed